docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -21,22 +21,55 @@ your other Uptime monitors.
 [role="screenshot"]
 image::images/synthetic-app-overview.png[Synthetics app overview]
 
-Select a specific journey to expand it.
-The following details are provided for each step:
+[[analyze-synthetic-journeys]]
+== Analyze synthetic journeys
 
-* Step name
-* Step script
-* A screenshot taken during the step
+On the *Monitors* table, you can click a listed journey to analyze it further
+on the monitor *Overview* page. You can examine journey details, including its
+availability, monitor ID, service availability per monitoring location, the timing
+for each synthetic check performed, and check statuses over time.
+
+The *History* table lists the total count of all the synthetic checks that completed or failed
+during the journey. The timestamp for each check is also displayed, along with its duration.
+For any failed checks, the error and the failed step are displayed.
+
+[role="screenshot"]
+image::images/synthetics_overview.png[Synthetics overview page]
+
+[[review-synthetic-checks]]
+== Review synthetic checks
+
+To drill down and review the steps performed by a synthetic check, select it in the *History*
```

Comment:
Not sure what the style guide says on this one

```suggestion
To drill-down and review the steps performed by a synthetic check, select it in the *History*
```

Reply:
It states drill down.

Reply:
I hate it 😄

Reply:
Me too!